### PR TITLE
Contributing

### PR DESCRIPTION
**Pull request by paul-krohn**
Skeleton contribution guidelines.

### CODE REVIEW

**Comment by crccheck** [comment]
The git history of this branch looks messy. Can you squash or rebase some of merge commits out? This is a feature branch, so I don't mind force pushing here.

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+1. Optional: create a virtual environment:
+   ``virtualenv cw2g && . cw2g/bin/activate``
+2. Install requirements: ``pip install -r requirements.txt``
+3. Run the test suite: ``make test``
+4. Verify the tests pass over all supported Python versions: ``tox``
+
+Pull requests
+~~~~~~~~~~~~~
+
+Standard-issue github project flow, summarized: 1. Fork the report 2.
```
Comment (crccheck) [inline CONTRIBUTING.rst]: typo s/report/repo

This would look better as a list too like:

1. Milk
2. Oreos
3. Ice cream

File: tox.ini [hunk 2]
```diff
@@ -4,8 +4,9 @@ envlist = py27, py34
 [testenv]
 commands = {envpython} -m unittest discover
 deps =
```
Comment (crccheck) [inline tox.ini]: We should probably just link this with requirements.txt Originally, I was obsessed with having a minimal list of requirements for every environment, but now I think the technical debt of maintaining those isn't worth it.

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+Create a branch 3. Check in your changes 4. Create a pull request 5. An
+existing contributor will do a code review, and as applicable ask for
+changes, mark the pull request with a :+1:, or merge it in, bump the
+version number, and cut/tag a release.
+
+Reporting Bugs
+~~~~~~~~~~~~~~
+
+This section guides you through submitting a bug report for Atom.
```
Comment (crccheck) [inline CONTRIBUTING.rst]: typo this is Buttwatch, not Atom.

File: CONTRIBUTING.rst (hunk 1 above)
Comment (paul-krohn) [inline CONTRIBUTING.rst]: Ugh, missed this in the output from the md->rst conversion tool I used.

**Comment by paul-krohn** [comment]
Unfortunately, squash/rebase on merge commits from another origin is beyond my git-fu.

**Pull request merged**